### PR TITLE
add vercheck

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -36,6 +36,9 @@ inputs:
   token:
     description: "GitHub token passed to the action"
     default: ${{ github.token }}
+  skip-check-for-update:
+    description: 'Set true to skip check for update functionality'
```
Comment (yonidavidson) [inline action.yml:40]:
```suggestion
    description: 'Set true to skip check for update'
```

File: src/vercheck.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+export interface Vercheck {
+  Version?: string
+  Summary?: string
+  Link?: string
+  Advisory?: string
+}
+
+export async function vercheck(cur: string): Promise<Vercheck> {
+  const c = new HttpClient('atlas-action')
+  const resp = await c.get(`https://vercheck.ariga.io/atlas-action/${cur}`)
```
Comment (yonidavidson) [inline src/vercheck.ts:12]: are you adding test=1 in case it's a local testing run?

**Review by yonidavidson — APPROVED** [review]
A few comments - looks good

File: src/vercheck.ts (hunk 2 above)
Comment (rotemtam) [reply src/vercheck.ts:12]: i'm using `nock` to mock the backend

**Review by giautm — APPROVED** [review]

**Pull request merged**